haas/ext/switches/migrations/mock/b5b31d19257d_rename_mockswitch_table_for_flask_.py

```diff
@@ -7,17 +7,16 @@
 Create Date: 2016-03-22 05:11:19.585905
 
 """
+from alembic import op
```

Comment:
It's probably fine, but there's a template somewhere in the alembic directory that is generating these, so it's going to keep generating things that violate the conventions if we don't modify it. If we rig up the test infrastructure it will yell at us the next time someone adds a migration script, and I'm frankly fine just dealing with it then.